<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1871</width>
      <height>871</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>530</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string>SVJETLINA KARTICE</string>
      </property>
     </widget>
     <widget class="QSlider" name="brightSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>580</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="sliderPosition">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="brightLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>650</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 190, 111);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string>malim slovima i sve zajedno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputID">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string>eventID:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteID">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Izbriši eventID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addID">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dodaj eventID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="albumButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>510</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lokacija albuma</string>
      </property>
     </widget>
     <widget class="QLabel" name="albumLabel">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>530</y>
        <width>761</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cardLabel">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>380</y>
        <width>491</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="karticaButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>370</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Odaberi karticu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QCheckBox" name="WhatsAppCheckBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>WhatsApp</string>
      </property>
     </widget>
     <widget class="QLabel" name="arduinoLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(246, 97, 81);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="arduinoTest">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Test connection</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ArduinoCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Arduino</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(165, 29, 45);</string>
      </property>
      <property name="text">
       <string>Provjeri USB kabel ako ne radi (uštekaj/ištekaj)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="promotivnaButton">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>390</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Printaj 4 promotivne</string>
      </property>
     </widget>
     <widget class="QPushButton" name="slideshowButton">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>120</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Slideshow</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>930</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>START BJUTN</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
